assets/quill-cursors.scss

```diff
@@ -50,6 +50,12 @@ $transition-func: cubic-bezier(0.250, 0.460, 0.450, 0.940);
   position: relative;
   // Make sure we don't show cursors/selection rectangles outside the editor
   overflow: hidden;
+
+  &.touch-device {
+    .ql-cursor-caret-container {
+      z-index: -1;
```

Comment:
No, it doesn't. We need it, because if we don't add it, then we can't «move» event under the caret, because it overlays the text. To make it work, I've added `touchstart` [listener](https://github.com/reedsy/quill-cursors/pull/80/files#diff-af5f5d3ad55cd5396de2dfe15f1c70653ade553478003b1eb087a446028db3d1R117), which detects near cursor and shows the flag (and it's closed after delay).